src/bms_main.cpp

```diff
@@ -59,6 +59,8 @@ void bms_setup() {
   // TODO(everyone): Experiment heavily on what is the smallest value accepted
   // by different boards without throwing the error.
   relay.setCurrentRewriterCallback([](float amps) {
+    // coefficient is multiplied by 2 because the reported current is scaled by
+    // .05
```

Comment:
Nope, all the scaling is happening in the background, transparently to this callback. You just gotta transform the value passed to you, no other adjustments needed.